.github/workflows/changelog.yml

```diff
@@ -3,6 +3,12 @@ name: Show changelog since last release
 on:
   workflow_dispatch:
 
+# cancel eventually still running jobs for the same branch and workflow
+# to save resources.
+#
+# NOTE:
+# This *can* lead to the effect that the first job that fails due to an
+# error introduced by a commit is *not* the run for the same commit but a later one.
```

Comment:
Sorry I don't get the meaning of the comment and it's consequences. Personally I also don't need these comments in the code and a note in the PR would be enough.

Reply:
thanks for the feedback.
i added these because I was worried that canbcelled runs can lkead to confusion about which commit really introduced a change that leads to an error. 
But  if the explanation is harder to understand than the corner case it should warn about, i can remove it.

Reply:
I have removed these comments